xero-app-store.yml

```diff
@@ -0,0 +1,260 @@
+                              "name": "Small",
+                              "status": "ACTIVE",
+                              "subscriptionItems": [
+                                {
+                                  "endDate": "2021-07-20T03:13:48Z",
+                                  "id": "c7336bf6-8a47-4f13-9fc0-82420e6922c8",
+                                  "price": {
+                                    "amount": 50,
+                                    "currency": "AUD",
+                                    "id": "31acefbe-bdb7-4329-84d6-51e9afd95327"
+                                  },
+                                  "product": {
+                                    "id": "56d66073-ff78-497b-a726-ca9d56fdafa3",
+                                    "name": "Small",
+                                    "type": "FIXED"
+                                  },
+                                  "startDate": "2021-07-20T03:13:48Z"
+                                }
+                              ]
+                            }
+                          ],
+                          "startDate": "2021-07-20T03:13:48Z",
+                          "status": "ACTIVE"
+                        }
+                      '
+        "404":
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/ProblemDetails"
+          description: "When a failure occurs in the endpoint"
+      security:
+        - OAuth2: [marketplace.billing]
+components:
+  securitySchemes:
+    OAuth2:
+      type: oauth2
+      description: For more information
+      flows:
+        authorizationCode:
+          authorizationUrl: "https://login.xero.com/identity/connect/authorize"
+          tokenUrl: "https://identity.xero.com/connect/token"
+          scopes:
+            marketplace.billing: Grant read-only access to fixed assets
+  parameters:
+    subscriptionId:
+      required: true
+      in: path
+      name: subscriptionId
+      x-snake: subscription_id
+      description: Unique identifier for Subscription object
+      example: "00000000-0000-0000-0000-000000000000"
+      x-example-java: UUID.fromString("00000000-0000-0000-0000-000000000000")
+      x-example-php: '&quot;00000000-0000-0000-0000-000000000000&quot;'
+      x-example-csharp: Guid.Parse("00000000-0000-0000-0000-000000000000");
+      schema:
+        type: string
+        format: uuid
+  schemas:
+    Subscription:
+      type: object
+      properties:
+        currentPeriodEnd:
+          description: |
+            End of the current period that the subscription has been invoiced for.
+          format: date-time
+          type: string
+        endDate:
+          description: If the subscription has been canceled, this is the date when the subscription ends. If null, the subscription is active and has not been cancelled
+          format: date-time
```

Comment:
I haven't tested this API yet. Do we know for sure that we return a simple date-time format and not the MS date format used in our other APIs (ex. "/Date(1476316800000+0000)/")?

Reply:
It does look that way. In newer API's I have not seen the ms date pattern, but would be good to know from the core eng team.

@sethreidnz 

<img width="303" alt="Screen Shot 2021-08-04 at 11 21 39 AM" src="https://user-images.githubusercontent.com/2935387/128225861-3251e0f9-f4f3-4b08-ad36-bce05b9c5d62.png">